### DISCUSSION TITLE
podman system service command not found in v3.4.4

### DISCUSSION THREAD

**Discussion by antonio-salgado-tfs** [containers/podman#13207] [Q&A] [answered]
Hi, I recently migrated from docker to podman, I'm trying to start the service to be able to communicate with API (the one compatible with docker)

I'm using this command `podman system service --time 5` but I'm getting this error `Error: unknown flag: --time`
according to this doc: https://docs.podman.io/en/v3.4.4/markdown/podman-system-service.1.html

lastly I notice that in the help command for `podman system` and the `service` command is not listed, so my questions are:

- is podman system service available in v3.4.4?
- if not, what version should I be using to start the service for API calls?

Thanks, 🙂

podman version: 3.4.4
OS: macOS bigsur 11.6

---
```sh
podman system --help
```
```
Manage podman

Description:
  Manage podman

Usage:
  podman system [command]

Available Commands:
  connection  Manage remote ssh destinations
  df          Show podman disk usage
  info        Display podman system information
  prune       Remove unused data
```

**Comment by rhatdan** [comment, marked as answer]
Podman on a Mac is podman-remote, it is not the same as the full blown Podman.

The full Podman is running within the podman machine (VM) And it has `podman system service`

**Reply by antonio-salgado-tfs** [reply]
thanks @rhatdan  for the information! 😀
